config/GregTech/GregTech.cfg

```diff
@@ -364,6 +415,63 @@ undergroundfluid {
 
     }
 
+    io {
+        # Dimension ID or Class Name
+        S:Dimension=Io
+
+        sulfuric_acid {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=80
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=5
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount=350
+
+            # Min amount generation (per operation, sets the VeinData) 0 MIN
+            I:MinAmount=0
+
+            # Fluid registry name
+            S:Registry=sulfuricacid
+        }
+
+        carbon_dioxide {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=80
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=5
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount=750
+
+            # Min amount generation (per operation, sets the VeinData) 0 MIN
+            I:MinAmount=0
+
+            # Fluid registry name
+            S:Registry=carbondioxide
+        }
+
+        phosphoric_acid {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=80
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=5
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount=800
```

Comment:
800 is too high for a pure substance, that has direct use

Reply:
changed to 400

Reply:
need to be tested why it not worked in the first place